wbooks/src/app/components/CustomButton/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { TouchableOpacity, Text } from 'react-native';
+
+import styles from './styles';
+
+interface Props {
+  label: string;
+  type: string;
+}
+
+function CustomButton({ label, type }: Props) {
+  return (
+    <TouchableOpacity style={type === 'primary' ? styles.primaryContainer : styles.secondaryContainer}>
```

Comment:
Instead of having a `primary`  and `secondary` you should have a base style, for example, `primary` and for `secondary` only change the styles

Also, instead of prop `type` you should have `secondary?: boolean`:
```
 <TouchableOpacity style={[styles.primaryContainer, secondary && styles.secondaryContainer]}>
```